frontend/packages/dev-console/src/components/import/ImportStrategySelector.tsx

```diff
@@ -0,0 +1,85 @@
+import * as React from 'react';
+import { FormGroup, Grid, GridItem, Tile } from '@patternfly/react-core';
+import { LayerGroupIcon, CubeIcon, GitAltIcon, StarIcon } from '@patternfly/react-icons';
+import { FormikValues, useField, useFormikContext } from 'formik';
+import { useTranslation } from 'react-i18next';
+import { ImportStrategy } from '@console/git-service/src';
+import { getFieldId } from '@console/shared/src';
+import { DetectedStrategyFormData } from './import-types';
+import './ImportStrategySelector.scss';
+
+const ImportStrategySelector: React.FC = () => {
+  const { t } = useTranslation();
+  const {
+    values: {
+      import: { recommendedStrategy },
+    },
+    // setFieldValue,
+    // setFieldTouched,
+  } = useFormikContext<FormikValues>();
+  const [selected] = useField<DetectedStrategyFormData>('import.selectedStrategy');
```

Comment:
You can directly take this out from value above. No need to use `useField` if you just need the value.